internal/cmd/create/cluster/create_cluster_cmd.go

```diff
@@ -47,9 +47,10 @@ var templatesFS embed.FS
 func Cmd() *cobra.Command {
 	runner := &runnerContext{}
 	result := &cobra.Command{
-		Use:   "cluster [flags]",
-		Short: "Create a cluster",
-		RunE:  runner.run,
+		Use:     "cluster [flags]",
+		Aliases: []string{"fulfillment.v1.Cluster"},
```

Comment:
Can we do something to avoid hard-coding this string here? For example:

```go
Aliases: []string{
       proto.MessageName((*ffv1.Cluster)(nil))
}
```

That way we will not need to explicitly remember changing this if we ever change the name of the type, and we avoid mistakes.